truss/templates/tgi/proxy.conf.jinja

```diff
@@ -20,6 +21,7 @@ server {
     # Predict
     location ~ ^/v1/models/model:predict$ {
         proxy_redirect off;
+        proxy_read_timeout 300s;
```

Comment:
In both of these places, could you leave this as a comment:

```
We use the proxy_read_timeout directive here (instead of proxy_send_timeout) as it sets the timeout for reading a response from the proxied server vs. setting a timeout for sending a request to the proxied server.
```